iothub_client/src/iothub_client_core.c

```diff
@@ -19,6 +19,9 @@
 #include "azure_c_shared_utility/xlogging.h"
 #include "azure_c_shared_utility/singlylinkedlist.h"
 #include "azure_c_shared_utility/vector.h"
+#include "iothub_client_options.h"
+
+#define LOOP_TIMEOUT_DEFAULT 1
```

Comment:
A good name will have the terms FREQUENCY + DEFAULT + IN_MS (or MILLISEC).
Maybe DEFAULT_DO_WORK_FREQUENCY_MILLISEC ?

Reply:
addressed